TabloidMVC/Views/Tags/Index.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model IEnumerable<TabloidMVC.Models.Tags>
+
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h1>Index</h1>
+
+<p>
+    <a asp-action="Create">Create New</a>
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.Id)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Name)
+            </th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+@foreach (var item in Model) {
+        <tr>
+            <td>
+                @Html.DisplayFor(modelItem => item.Id)
```

Comment:
The ticket says we need to show tag name-- we don't need to show the user the tag's primary key, and we usually don't-- database keys aren't helpful to the end user. Let's lose this column